actions.md

```diff
@@ -143,7 +143,8 @@ These are the primary actions that a user of libaktualizr can perform through th
       - [x] Set new version to pending status after an OSTree update trigger (aktualizr_test.cc)
       - [x] Send EcuInstallationAppliedReport to server after an OSTree update trigger (aktualizr_test.cc)
       - [x] Uptane check for updates and manifest sends are disabled while an installation is pending reboot (aktualizr_test.cc)
-      - [ ] Trigger a system reboot if `reboot_after_install` is set in the configuration
+      - [ ] Trigger a system reboot at the end of the installation process in case of the ostree package manager usage (OTA-2135)
+      - [x] Emulate a reboot at the end of the installation process in case of the fake package manager usage (OTA-2135, aktualizr_test.cc)
```

Comment:
BTW In the past I only added ticket numbers for Requirement tickets. It seems like no one is very interested in those, though, and I'm hesitant to advocate for more Jira bureaucracy, so that hasn't gone anywhere in a while. There's no harm in keeping the ticket number for the task here, but generally you probably don't need to worry about that.